css/main.css

```diff
@@ -1,3 +1,733 @@
+}
+
+/* HEADER-SEARCH-BAG */
+
+.header-search-bag {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background-color: var(--white);
+  padding: 1.4rem 1rem;
+}
+
+.bag {
+  width: 1.5rem;
+  height: 1.9rem;
+  background-image: url("/home/marina/Desktop/CodeYouFuture/responsive-grid-project/img/bag.svg");
+  background-size: cover;
+  filter: invert(43%) sepia(6%) saturate(4186%) hue-rotate(329deg)
+    brightness(89%) contrast(91%);
+}
+
+.search-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  outline: none;
+  position: relative;
+  /* direction: rtl; rigth to left */
+  width: 100%;
+  background-color: var(--white);
+  color: var(--red_brown);
+  padding: 0 1.3rem;
+}
+
+.search-container input:hover {
+  border: 1px solid var(--red_brown);
+}
+
+.search-container input {
+  height: 2.5rem;
+  padding: 0 1.3rem;
+  border: none;
+  background: var(--white);
+}
+
+.search-container input::placeholder {
+  color: var(--red_brown);
+  font-size: 1.2rem;
+  font-weight: 100;
+  padding-left: 1rem;
+}
+
+.search-icon img {
+  position: absolute;
+  width: 17px;
+  height: 16px;
+  top: 12px;
+  left: 26px;
+  filter: invert(38%) sepia(49%) saturate(567%) hue-rotate(329deg)
+    brightness(92%) contrast(89%);
+}
+
+/* HEADER-LOGO-TITLE-MENU-BURGER */
+.header-logo_title_menu_burger {
+  padding: 1rem 1rem 0 1rem;
+  margin: 2rem 0 1.5rem 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo_title {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.logo_title h1 {
+  font-family: var(--heading_font-family);
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.logo_title p {
+  font-size: 1.1rem;
+  font-weight: 600;
+}
+
+rect {
+  fill: var(--red_brown);
+  border-radius: 2px;
+}
+
+/* MAIN */
+.main {
+  grid-area: main;
+  text-align: center;
+}
+
+.hero {
+  height: 35rem;
+}
+
+.hero-image {
+  width: 100%;
+  height: 35rem;
+  background-image: url("../img/mobile-big-img.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.hero-content {
+  position: relative;
+  bottom: 33rem;
+  color: var(--white);
+}
+
+.logo-icon-words {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+  margin-top: 2rem;
+}
+
+.logo-icon img {
+  height: 10rem;
+  filter: invert(100%) sepia(4%) saturate(139%) hue-rotate(198deg)
+    brightness(117%) contrast(100%);
+}
+
+.logo-words {
+  border-bottom: 2px solid var(--white);
+  width: 4rem;
+  margin-bottom: 2.3rem;
+}
+
+.logo-words p {
+  font-size: 1rem;
+  font-weight: bolder;
+  letter-spacing: 0.5rem;
+  margin-bottom: 0.4rem;
+}
+
+.hero-content h1 {
+  font-size: 2.7rem;
+  margin-bottom: 1rem;
+  margin-top: 0.7rem;
```

Comment:
The font-family needs defining here, it's using the wrong one.

Reply:
the font Fjalla One doesn't work.